vm/src/stdlib/socket.rs

```diff
@@ -409,6 +410,17 @@ fn socket_inet_ntoa(packed_ip: PyBytesRef, vm: &VirtualMachine) -> PyResult {
     Ok(vm.new_str(Ipv4Addr::from(ip_num).to_string()))
 }
 
+fn socket_htonl(host: PyIntRef, vm: &VirtualMachine) -> PyResult {
+    if host.as_bigint().sign() == Sign::Minus {
+        return Err(
+            vm.new_overflow_error("can't convert negative value to unsigned int".to_string())
+        );
+    }
+
+    let host = host.as_bigint().to_u32().unwrap();
+    Ok(vm.new_int(host.to_be()))
```

Comment:
Is there a rust library with the `htonl` and `ntohl` functions?

Reply:
I found `htonl` and `ntohl` in [socket](https://docs.rs/socket/0.0.7/socket/) crate. but it is not very popular.

Reply:
Okay, maybe stick to this version, until we find a good socket crate. @palaviv do you have any ideas?

Reply:
Even from [gnu libc](https://github.com/lattera/glibc/blob/895ef79e04a953cac1493863bcae29ad85657ee1/inet/htonl.c#L24-L34), it essentially is this simple of an implementation. Rust's [`u32::to_be`](https://doc.rust-lang.org/src/core/num/mod.rs.html#2616-2625) looks basically the same to that libc `htonl`.